.mergify.yml

```diff
@@ -143,11 +143,11 @@ pull_request_rules:
       - 'check-success=encryption-pvc-db-wal'
       - 'check-success=encryption-pvc-kms-vault-token-auth'
       - 'check-success=TestCephSmokeSuite (v1.15.12)'
-      - 'check-success=TestCephSmokeSuite (v1.21.0)'
+      - 'check-success=TestCephSmokeSuite (v1.20.2)'
       - 'check-success=TestCephHelmSuite (v1.15.12)'
-      - 'check-success=TestCephHelmSuite (v1.21.0)'
-      - 'check-success=TestCephMultiClusterDeploySuite (v1.21.0)'
-      - 'check-success=TestCephUpgradeSuite (v1.21.0)'
+      - 'check-success=TestCephHelmSuite (v1.20.2)'
```

Comment:
In master we are running against 1.21, but it release-1.6 we are running against 1.20. Is there any way we can make this more generic so we're not dependent on different versions in different branches?

Reply:
We need to change the name of the job to be fixed instead of being generated by the matrix.